src/controller/SnoozeForm.tsx

```diff
@@ -154,26 +155,26 @@ class SnoozeForm extends Component<Props, State> {
     const snoozeStart = new Date(
       this.props.rowData.snoozeInformation.snoozeStart
     ).toLocaleDateString("en-US");
-
     return (
-      <div className="usa-alert usa-alert--warning usa-alert--slim">
-        <div className="usa-alert__body">
-          <p className="usa-alert__text">
-            Case was previously snoozed on {snoozeStart}. Reason given:{" "}
-            {this.props.rowData.snoozeInformation.snoozeReason}.
-          </p>
-        </div>
-      </div>
+      <UsaAlert
+        alertType="warning"
+        text={
+          <React.Fragment>
+            Originally triaged on: {snoozeStart}.<br />
```

Comment:
I would advocate for "previously", rather than "originally"—we don't have the entire history here.